example/index.html

```diff
@@ -109,7 +109,7 @@
         }
       }
 
-      var token = 'YOUR USER TOKEN';
+      var token = 'ORG_ACCESS_TOKEN';
 
       var sfxClient = signalfx.streamer.SignalFlow(token);
```

Comment:
Does the user type the word "token" here? If not, we need to use the appropriate format for this kind of statement.

Reply:
yes, they use the word token here.